deanonimize getrequest goroutine

hellt opened the pull request:
deanonimized GetRequest goroutine. The output has not been changed

karimra commented:
`prototext.MarshalOptions{Multiline: false}.Format(req)` doesn't print the whole message ( doesn't include encoding, models, extension,...)
So either keep the old one, or add the missing parts manually on top of `prototext.Format`

hellt commented:
I've reverted the commit that introduced prototext marshaller and [asked](https://github.com/openconfig/gnmi/issues/78) the community what would be the best way to fully marshal the gnmi go struct to prototext

Also renamed the method to getRequest
